homeassistant/helpers/collection.py

```diff
@@ -555,20 +554,19 @@ def async_setup(
                 ),
             )
 
-        if create_create:
-            websocket_api.async_register_command(
-                hass,
-                f"{self.api_prefix}/create",
-                websocket_api.require_admin(
-                    websocket_api.async_response(self.ws_create_item)
-                ),
-                websocket_api.BASE_COMMAND_MESSAGE_SCHEMA.extend(
-                    {
-                        **self.create_schema,
-                        vol.Required("type"): f"{self.api_prefix}/create",
-                    }
-                ),
-            )
+        websocket_api.async_register_command(
```

Comment:
We are now registering a WS command that is not being used. It doesn't take up much space, but 🤷

Reply:
Yes, it's a bit silly, but the goal of this PR is to simplify `StorageCollectionWebsocket`.
I think the gain of simplifying the base class is larger than the few bytes wasted by a non-working WS command.